fix(migration): Warn about not stopping BPM services before migrating. (#1441)
bonitasoft/bonita-doc: md/migrate-from-an-earlier-version-of-bonita-bpm.md

   | db.password    | The password used to authenticate to the database                | bpm                                                        |

    ::: info
    Note: If you are using MySQL, add `?allowMultiQueries=true` to the URL. For example,
    `db.url=jdbc:mysql://localhost:3306/bonita_migration?allowMultiQueries=true`.  
    Also, if you are migrating to Bonita 7.9+, you must upgrade your database server to MySQL 8.0, see [Migrating to Bonita 7.9+ using MySQL](#mysql8) specific procedure below.
    :::

6. If you use a custom Look & Feel, [export](managing-look-feel.md) it, and then [restore the default Look & Feel](managing-look-feel.md).
7. It is generally not recommended to pause the BPM services before migrating. The next point being the exception.
8. If you use a Business data model and migrate, there are a few things to keep in mind:
    * If you are migrating to a version that requires the BDM to be redeployed, or a version above that (see [above](#bdm_redeploy)), you will need to redeploy your BDM after your migration.

>     * In this case, and only in this case, it is recommended to stop your BPM services before migrating, so as to be able to redeploy your BDM immediately after migration (and not having to deal with eventual errors in automated processes as you restart your platform after migration).

Reviewer: ```suggestion
    * In this case, and only in this case, you have to stop your BPM services before migrating, so as to be able to redeploy your BDM immediately after migration (and not having to deal with eventual errors in automated processes as you restart your platform after migration).
```
